Loops_and_set/task_1.py

```diff
@@ -0,0 +1,5 @@
+first = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
+second = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13]
+
+uniq_elements = list(set(first + second))
```

Comment:
here should be .intersection() method, not union()

Reply:
![image](https://github.com/MykhailoTsyba/Homework/assets/51053483/46e8ab02-38dd-4591-8abf-b2b985c7e76f)